mite/cli/test.py

```diff
@@ -153,6 +153,10 @@ def test_scenarios(test_name, opts, scenarios, config_manager):
     # Run one last report before exiting
     controller.report(receiver.recieve)
     has_error = http_stats_output is not None and http_stats_output.error_total > 0
+
+    # Ensure any open files get closed
+    del receiver._raw_listeners
```

Comment:
```suggestion
    del receiver
```

I think it should be like this, to more robustly ensure all resources are cleaned up (the alternative would be to also `del receiver._listeners`)

Reply:
With just `del receiver` it never cleans up the Collector, so I just did your alternative

Reply:
hmm, odd.  I wonder why :thinking: although it doesn't matter _that_ much.